test/unit/view.js

```diff
@@ -151,101 +247,85 @@ describe('View class', () => {
           },
         };
         view.append(div);
+        assert.calledOnce(view.iframe.document.body.appendChild);
         assert.calledWith(view.iframe.document.body.appendChild, div);
       });
 
-      it('appends to node if no iframe', () => {
+      it('appends to node if iframe does not exist', () => {
         const div = document.createElement('div');
-        const appendChildSpy = sinon.spy(view.component.node, 'appendChild');
+        const appendChildStub = sinon.stub(view.component.node, 'appendChild');
         view.iframe = null;
         view.append(div);
-        assert.calledWith(appendChildSpy, div);
+        assert.calledOnce(appendChildStub);
+        assert.calledWith(appendChildStub, div);
+        appendChildStub.restore();
       });
     });
 
     describe('destroy()', () => {
       it('removes node from parent', () => {
-        const component = new Component({
-          id: 1234,
-        }, {browserFeatures: {}});
-        const view = new View(component);
         view.node = {
           parentNode: {
             removeChild: sinon.spy(),
           },
         };
         view.destroy();
         assert.calledOnce(view.node.parentNode.removeChild);
+        assert.calledWith(view.node.parentNode.removeChild, view.node);
       });
     });
 
     describe('renderChild()', () => {
-      it('calls updateNode with node and html', () => {
-        const component = new Component({
-          id: 1234,
-          options: {
-            product: {
-              viewData: {
-                title: 'lol',
-              },
-            },
-          },
-        }, {browserFeatures: {}});
-        component.typeKey = 'product';
-        const view = new View(component);
+      it('updates node with new node and html created from class name and template params', () => {
+        const html = '<h1>test</h1>';
+        const node = document.createElement('div');
         const template = {
-          render: sinon.stub().returns('<h1>BUY MY BUTTONS lol</h1>')
+          render: sinon.stub().returns(html),
         };
-        view.updateNode = sinon.spy();
-        view.wrapper = document.createElement('div');
-        const childNode = document.createElement('div');
-        childNode.className = 'title';
-        view.wrapper.appendChild(childNode);
+        view.wrapper = {querySelector: sinon.stub().returns(node)};
```

Comment:
Can we format this like `template` above, and also move out the stubs into their own variables for consistency?